web/timezone_views.py

```
@@ -0,0 +1,31 @@
+    timezone_name = request.POST.get("timezone")
+
+    if not timezone_name:
+        return JsonResponse({"status": "error", "message": "No timezone provided"}, status=400)
+
+    # Validate the timezone
+    try:
+        pytz.timezone(timezone_name)
+    except pytz.exceptions.UnknownTimeZoneError:
+        return JsonResponse({"status": "error", "message": "Invalid timezone"}, status=400)
+
+from django.http import HttpRequest
```

Ruff (0.8.2)

Module level import not at top of file

(E402)